sonar-project.properties

```diff
@@ -4,5 +4,6 @@ sonar.language=typescript
 sonar.sources=src/main
 sonar.tests=src/test
 sonar.test.exclusions=test/coverage/**
+sonar.cpd.exclusions=src/main/routes/watch-mk.ts
 sonar.javascript.lcov.reportPaths=coverage/lcov.info
-sonar.exclusions=src/main/app.ts,src/main/development.ts,src/main/index.js,src/main/server.ts,src/main/modules/properties-volume/index.ts,src/main/assets/js/cookie.js,src/main/routes/health.ts
+sonar.exclusions=src/main/app.ts,src/main/development.ts,src/main/index.js,src/main/server.ts,src/main/modules/properties-volume/index.ts,src/main/assets/js/cookie.js,src/main/routes/health.ts,src/test/unit/routes/watch-mk.ts
```

Comment:
looks like u refactored the test - can this be removed from the exclusion now or was it still kicking up a stink?